### ISSUE TITLE
Feature Request: Have option for split up Avatar types

### ISSUE THREAD

**Issue by jackHedaya** [Sitebase/react-avatar#169]
I think developers should have the option of only calling in one Avatar type. For example, I am only using the custom providers but webpack is going to include all the code for Google, Facebook, etc, which causes dependency bloating.

**Comment by JorgenEvens** [comment]
Definitely something we should look at.

For the time being I think you could use [resolve.alias](https://webpack.js.org/configuration/resolve/#resolvealias) in your webpack config to reduce the amount of bloat in your bundle.

You would alias the sources you would like to remove to something like the file below.
```js
module.exports = function() {};
module.exports.isCompatible = function() { return false; }
```

**Comment by jackHedaya** [comment]
Should I make a pull request?

**Comment by JorgenEvens** [comment]
Feel free to do so, would be appreciated. 👍

**Comment by JorgenEvens** [comment]
Hi @jackHedaya,

I've just released [v3.7.0](https://github.com/Sitebase/react-avatar/milestone/33?closed=1) which added support for the [customized sources list](https://github.com/Sitebase/react-avatar#reducing-bundle-size)!

**Comment by jackHedaya** [comment]
Awesome, thanks!
